src/dex-ui/pages/dao/utils.ts

```diff
@@ -98,3 +106,31 @@ export function getProposalSteps(
   }
   return { steps, activeStep };
 }
+
+export function getProposalData(proposal: Proposal): string {
+  if (proposal.type === ProposalType.AddNewMember) {
```

Comment:
A `switch` statement can be used instead of multiple `if` statements when checking a single variable against multiple conditions to produce more readable code.

Reply:
Done.